core/papersize.lua

```diff
@@ -63,21 +63,26 @@ local papersize = {
 }
 
 setmetatable(papersize, {
-    __call = function (self, size)
-      local _, _, x, y = string.find(size, "(.+)%s+x%s+(.+)")
-      if x and y then
-        return { SILE.measurement(x):tonumber(), SILE.measurement(y):tonumber() }
-      else
-        size = string.lower(size:gsub("[-%s]+", ""))
-        if self[size] then
+  __call = function(self, size, landscape)
+    local _, _, x, y = string.find(size, "(.+)%s+x%s+(.+)")
+    if x and y then
+      return { SILE.measurement(x):tonumber(), SILE.measurement(y):tonumber() }
+    else
+      size = string.lower(size:gsub("[-%s]+", ""))
+      if self[size] then
+        if landscape then
+          self[size][1], self[size][2] = self[size][2], self[size][1]
+          return self[size]
+        else
           return self[size]
         end
       end
-      SU.error(string.format([[Unable to parse papersize '%s'.
+    end
+    SU.error(string.format([[Unable to parse papersize '%s'.
   Custom sizes may be entered with 'papersize=<measurement> x <measurement>'.
   Predefined paper sizes include: %s]],
-  size, table.concat(pl.tablex.keys(papersize), ", ")))
-    end
-  })
+      size, table.concat(pl.tablex.keys(papersize), ", ")))
+  end
+})
```

Comment:
More irrelevant formatting changes?

Reply:
Sorry, I didn't know how relevant this kind of change is, I'll pay attention to this detail in my next PRs.

Reply:
The change itself may be fine, like I said I'm fine with style improvements. They just need to be in isolated commits (and PRs) that can be checked by using a diff with ignore-whitespace enabled so they are easy to check—not mixed in with actual code refactoring, fixing, or features.